README.md

```diff
@@ -16,20 +16,142 @@ Capabilities provided by `caikit-nlp`:
 | Tokenization         | 1. `RegexSentenceSplitter`                | 1. Demo purposes only                                                                                                                                       |
 | Embedding         | [COMING SOON]                | [COMING SOON]                                                                                                                                       |
 
-### Getting Started
+## Getting Started
+
+### Notebooks
 
 To help you quickly get started with using Caikit, we have prepared a [Jupyter notebook](examples/Caikit_Getting_Started.ipynb) that can be run in Google Colab. Caikit-nlp is a powerful library that leverages prompt tuning and fine-tuning to add NLP domain capabilities to caikit.
 
+### Installation
+
+To install from git repo:
+
+```bash
+python -m venv .venv
+source .venv/bin/activate
+pip install git+https://github.com/caikit/caikit-nlp
+```
+
+### Bootstrapping models
+
+`caikit_nlp` can use  Hugging Face models, allowing for direct download and bootstrapping.
+
+For example, to use [google/flan-t5-small](https://huggingface.co/google/flan-t5-small):
+
+```python
+import os
+# The env var ALLOW_DOWNLOADS has to be set to allow model downloads before importing caikit_nlp
+os.environ['ALLOW_DOWNLOADS'] = "1"
+
+import caikit_nlp
+
+model_name = "google/flan-t5-small"
+model = caikit_nlp.text_generation.TextGeneration.bootstrap(model_name)
+model.save(f"{model_name}-caikit") # optionally save the model
+```
+
+### Serving models
+
+To serve models, the following basic configuration can be used:
+
+```yaml
+# config.yml
+runtime:
+  library: caikit_nlp
+  local_models_dir: ./models
+
+log:
+  formatter: pretty # optional: log formatter is set to json by default
+```
+
+Start the server:
+
+```bash
+env CONFIG_FILES=./config.yml python -m caikit.runtime
+```
+
+The model can now be queried at `localhost:8080` via http or at `localhost:8085` via grpc.
+
+For example, using the http server and using curl to send a POST request:
+
+```bash
+curl --json '{
+    "model_id": "flan-t5-small-caikit",
+    "inputs": "At what temperature does liquid Nitrogen boil?"
+}' localhost:8080/api/v1/task/text-generation
+```
+
+We get the following response:
+
+```json
+{
+  "generated_text": "74 degrees F",
+  "generated_tokens": 5,
+  "finish_reason": "MAX_TOKENS",
+  "producer_id": {
+    "name": "Text Generation",
+    "version": "0.1.0"
+  },
+  "input_token_count": 10,
+  "seed": null
+}
+```
```

Comment:
Later on we can also move these into the `examples` folder.. We can have 2 parts of the "examples", 1 related to notebooks etc and other related to servers including examples from here 🤔